### PR TITLE
Ft crud user

### PR DESCRIPTION
**Pull request by mwafrika**
# what does this pull request do ?

- [ ] CRUD users (only admin, managers and super admin will have access to these routes)

### CODE REVIEW

**Review by Irirwanirira — APPROVED** [review]

File: src/controllers/auth.js [hunk 1]
```diff
@@ -71,3 +83,252 @@ export const register = async (req, res) => {
...
+  try {
+    const user = await findUserById(id, {
+      companyId,
+    });
+
+    if (!user) {
+      return formatResponse(res, StatusCodes.NOT_FOUND, [], "User not found");
+    }
+
+    const hashedPassword = await hashPassword(password);
```
Comment (alainburindi) [inline src/controllers/auth.js]: Do you mean while updating the user the password always need to be provided?

File: src/database/models/index.js [hunk 2]
```diff
@@ -1,12 +1,12 @@
-/* eslint-disable */
+"use strict";
```
Comment (alainburindi) [inline src/database/models/index.js]: What's the use of this line?

**Review by alainburindi — CHANGES_REQUESTED** [review]

File: src/controllers/auth.js (hunk 1 above)
Comment (mwafrika) [reply src/controllers/auth.js]: I changed the way to do so.

File: src/database/models/index.js (hunk 2 above)
Comment (mwafrika) [reply src/database/models/index.js]: normally it says that the code should be executed in strict mode and also it was generated automatically when creating a new model

**Review by jefftrojan — APPROVED** [review]

**Review by ciraganenicole — APPROVED** [review]

**Pull request closed without merging**